### ISSUE TITLE
Maxwell needs to parse SQL inside /*!  */ "comments"

### ISSUE THREAD

**Issue by jpeyper** [zendesk/maxwell#158]
I have been seeing this error while evaluating Maxwell in a Fedora VM.
```
09:25:53,554 DEBUG SchemaChange - SQL_PARSE <- "CREATE DATABASE /*! IF NOT EXISTS */ error"
09:25:53,556 DEBUG SchemaChange - SQL_PARSE ->   (parse (statement (create_database CREATE DATABASE (name (id error)))) <EOF>)
com.zendesk.maxwell.schema.ddl.SchemaSyncError: Unexpectedly asked to create existing database error
	at com.zendesk.maxwell.schema.ddl.DatabaseCreate.apply(DatabaseCreate.java:25)
	at com.zendesk.maxwell.MaxwellReplicator.processQueryEvent(MaxwellReplicator.java:280)
	at com.zendesk.maxwell.MaxwellReplicator.getEvent(MaxwellReplicator.java:253)
	at com.zendesk.maxwell.MaxwellReplicator.work(MaxwellReplicator.java:89)
	at com.zendesk.maxwell.RunLoopProcess.runLoop(RunLoopProcess.java:31)
	at com.zendesk.maxwell.Maxwell.run(Maxwell.java:89)
	at com.zendesk.maxwell.Maxwell.main(Maxwell.java:95)
09:25:53,558 INFO  SchemaPosition - Storing final position: null
```
A similar error occurs at the table level.
```
10:24:04,086 DEBUG SchemaChange - SQL_PARSE <- "create table /*! if not exists */ `_work`.`cdctest3` (
  `id` int (10) unsigned not null auto_increment,
  primary key (`id`)
)"
10:24:04,090 DEBUG SchemaChange - SQL_PARSE ->   (parse (statement (create_table (create_table_preamble create table (table_name (db_name (name (id `_work`))) . (name (id `cdctest3`)))) (create_specifications ( (create_specification (column_definition (name (id `id`)) (data_type (signed_type int (length ( 10 )) (int_flags unsigned) (column_options (nullability not null)) (column_options auto_increment))))) , (create_specification (index_definition (index_type_pk primary key (index_column_list ( (index_columns (index_column (name (id `id`)))) ))))) )))) <EOF>)
com.zendesk.maxwell.schema.ddl.SchemaSyncError: Unexpectedly asked to create existing table cdctest3
	at com.zendesk.maxwell.schema.ddl.TableCreate.apply(TableCreate.java:45)
	at com.zendesk.maxwell.MaxwellReplicator.processQueryEvent(MaxwellReplicator.java:280)
	at com.zendesk.maxwell.MaxwellReplicator.getEvent(MaxwellReplicator.java:253)
	at com.zendesk.maxwell.MaxwellReplicator.work(MaxwellReplicator.java:89)
	at com.zendesk.maxwell.RunLoopProcess.runLoop(RunLoopProcess.java:31)
	at com.zendesk.maxwell.Maxwell.run(Maxwell.java:89)
	at com.zendesk.maxwell.Maxwell.main(Maxwell.java:95)
10:24:04,188 INFO  SchemaPosition - Storing final position: BinlogPosition[awmsphx90022-bin.000003:203414608]
10:24:04,189 DEBUG SchemaPosition - Writing binlog position to maxwell.positions: BinlogPosition[awmsphx90022-bin.000003:203414608]
```

**Comment by jpeyper** [comment]
@osheroff, Would there be any kind of ETA in a fix for this? Days? Weeks? Months?

**Comment by osheroff** [comment]
fixed now in v0.14.4

**Comment by jpeyper** [comment]
So quick, thanks :)
